### DISCUSSION TITLE
Upper and lower case for links

### DISCUSSION THREAD

**Discussion by tschiekdev** [singerdmx/flutter-quill#846] [General]
https://github.com/singerdmx/flutter-quill/blob/a00b7fead108313ff3bd3f187b4c925abfad735f/lib/src/models/rules/insert.dart#L333

Any chance you could tweak the regular expression a bit to make links case-insensitive?

E.g. will
"WWW.GOOGLE.DE"
or just
"Www.google.de"
not recognized as a link.

Would be nice. Thank you. :-)

**Comment by singerdmx** [comment]
Could you submit a PR?
Or just reply with the correct regex and I will do it.

**Comment by singerdmx** [comment]
Fixed in https://github.com/singerdmx/flutter-quill/commit/a51327ba6ff0d05cfeed69e0ecb69f2890b35047

**Comment by tschiekdev** [comment]
Thank you!!!
